lib/asn1/src/asn1ct.erl

```diff
@@ -1382,7 +1382,7 @@ is_asn1_flag(no_ok_wrapper) -> true;
 is_asn1_flag(optimize) -> true;
 is_asn1_flag(per) -> true;
 is_asn1_flag({record_name_prefix,_}) -> true;
-is_asn1_flag(undec_rec) -> true;
+is_asn1_flag(undec_rest) -> true;
```

Comment:
Maybe it make sense to cover this logic by some tests to avoid similar issues in the future.